pkg/services/featuremgmt/manager.go

```diff
@@ -16,15 +17,17 @@ var (
 type FeatureManager struct {
 	isDevMod        bool
 	restartRequired bool
-	allowEditing    bool
-	licensing       licensing.Licensing
-	flags           map[string]*FeatureFlag
-	enabled         map[string]bool // only the "on" values
-	config          string          // path to config file
-	vars            map[string]any
-	startup         map[string]bool   // the explicit values registered at startup
-	warnings        map[string]string // potential warnings about the flag
-	log             log.Logger
+
+	Settings setting.FeatureMgmtSettings
```

Comment:
@mmandrus -- the biggest change here is that the FeatureManager uses *the* settings object rather than a copy of it from settings.